assets/css/edd-admin.css

```diff
@@ -3616,6 +3616,10 @@ body.download_page_edd-reports {
 	background-color: #fcfcfc;
 }
 
+.edd-vertical-sections .section-nav:last-of-type {
```

Comment:
@robincornett This should be targeting `.section-title:last-of-type`. Right now it's putting margin on the whole list:

<img width="230" alt="Screen Shot 2020-04-29 at 3 06 18 PM" src="https://user-images.githubusercontent.com/491124/80636181-04fae600-8a2b-11ea-82ca-c34e0a0e75e3.png">
<img width="253" alt="Screen Shot 2020-04-29 at 3 06 24 PM" src="https://user-images.githubusercontent.com/491124/80636183-05937c80-8a2b-11ea-9df8-8d7fd1556753.png">